internal/peer/node/start.go

```diff
@@ -204,7 +204,7 @@ func serve(args []string) error {
 	grpc.EnableTracing = true
 
 	logger.Infof("Starting %s", version.GetInfo())
-
+	logger.Infof("Starting locally rebuilt: mbw01")
```

Comment:
Did you mean to leave this line in?

Reply:
No; I thought I'd removed that... thanks @andrew-coleman